docs/operation/cluster_and_service_configuration.rst

```diff
@@ -0,0 +1,64 @@
+Cluster and Services Configurations
```

Comment:
This title suggests we will find a procedure in the topic.
Looking at it, I only see tables and references.
I think it should be renamed "Cluster and Services Configuration Parameters" to avoid any confusion (we do not present any procedure here)

Reply:
True, we have a ticket scheduled for next week that will add a procedure to this section.
As a clue, I added it in the todo section of this doc.

As a result, please let us keep the topic